issue-tracker-ws/src/main/java/org/example/issuetracker/web/api/IssueController.java

```diff
@@ -67,7 +67,7 @@ public IssueController(IssueService issueService) {
     }
 
     private Predicate<Issue> getIssueWithStatus(String status) {
-        return issue -> issue.getStatus().equals(IssueStatus.valueOf(status.toUpperCase()));
+        return issue -> issue.getStatus().equals(IssueStatus.valueOf(status.toUpperCase().replaceAll("\\s", "_")));
```

Comment:
oui ça marche mais la solution est trop simpliste!

Reply:
Ah

Reply:
l'idée c'est plutôt de remettre la propriété String que tu avais ajouté dans IssueStatus et de retrouver la bonne valeur énumérée (IN_PROGRESS) en passant "In Progress" à une méthode qui va faire le lien entre le libellé "In Progress" et la valeur énumérée IN_PROGRESS

Reply:
Hum.. D'accord. Je vais essayer de pleurer encore quelques heures sur le sujet alors :)